runtime/src/main/java/com/dylibso/chicory/runtime/Module.java

```diff
@@ -61,15 +61,24 @@ private Module(
         this.start = start;
         this.typeValidation = typeValidation;
         this.module = validateModule(module);
-        this.exports = new HashMap<>();
-        int cnt = module.exportSection().exportCount();
+        this.exports = genExports(module.exportSection());
+    }
+
+    private static HashMap<String, Export> genExports(ExportSection export) {
```

Comment:
Nit: we could change `exports` to be declared as `Map` instead of `HashMap`, then this could return `Map`

Reply:
right! done